dist/bin/current/roddy.sh

```diff
@@ -129,7 +128,25 @@ elif [[ "$parm1" == "createworkflow" ]]; then
 fi
 
 IFS=""
-#[[ $RMIPORT != "" ]] && export DBG_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005" && echo "Opened up rmi debugger port"
-java $DBG_OPTS -cp .:$libraries:${RODDY_BINARY} de.dkfz.roddy.Roddy $*
-# -verbose:class
+export RODDIES_GROOVYLIB_PATH=`readlink -f ${RODDY_BINARY_DIR}/lib/groovy*.jar`
+
+debuggerSettings=""
+[[ -n ${DEBUG_ROODY} ]] && debuggerSettings=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+caller=$(checkAndDownloadGroovyServ "${RODDY_DIRECTORY}")
+
+echo "RGL: "$RODDIES_GROOVYLIB_PATH
```

Comment:
Unconventional to use the Genitive here. Think of grepping for all Roddy variables! Would be harder like this.